src/request.rs

```diff
@@ -42,6 +43,15 @@ pub struct AppendOperation {
     requester: Requester,
 }
 
+/// Wrapper message that contains the request, message ID, and the requester ID,
+/// which should also contain the signature(s) if it's sent by the owner(s).
```

Comment:
"signature(s)" → "signature" since we won't aggregate the sigs - that'll be done before sending the request.